build.gradle.kts

```diff
@@ -43,6 +43,14 @@ qodana {
 }
 
 tasks {
+    compileKotlin {
```

Comment:
I need to verify if this block of code is even needed. I copied it from
https://plugins.jetbrains.com/docs/intellij/kotlin.html#kotlin-standard-library to address some testing errors

Reply:
Unneeded afterall.